test-app/src/message-board.ts

```diff
@@ -1,668 +1,675 @@
-import * as path from "path";
-import * as appsync from "@aws-cdk/aws-appsync-alpha";
-import {
-  App,
-  aws_dynamodb,
-  RemovalPolicy,
-  Stack,
-  aws_events,
-  Duration,
-} from "aws-cdk-lib";
-import {
-  $AWS,
-  $SFN,
-  $util,
-  AppsyncResolver,
-  Function,
-  StepFunction,
-  Table,
-  EventBus,
-  Event,
-  ExpressStepFunction,
-  AppsyncField,
-  AppsyncContext,
-} from "functionless";
+// import * as path from "path";
+// import * as appsync from "@aws-cdk/aws-appsync-alpha";
+// import {
+//   App,
+//   aws_dynamodb,
+//   RemovalPolicy,
+//   Stack,
+//   aws_events,
+//   Duration,
+// } from "aws-cdk-lib";
+// import {
+//   $AWS,
+//   $SFN,
+//   $util,
+//   AppsyncResolver,
+//   Function,
+//   StepFunction,
+//   Table,
+//   EventBus,
+//   Event,
+//   ExpressStepFunction,
+//   AppsyncField,
+//   AppsyncContext,
+// } from "functionless";
```

Comment:
What happened in this file? Looks like it was nuked?

Reply:
woops.